lib/pbench/test/unit/server/test_upload.py

```diff
@@ -920,11 +928,15 @@ def test_upload_archive(self, client, pbench_drb_token, server_config, tarball):
         }
 
     @pytest.mark.freeze_time("1970-01-01")
-    def test_upload_nometa(self, client, pbench_drb_token, server_config, tarball):
+    def test_upload_nometa(
+        self, client, pbench_drb_token, server_config, tarball, monkeypatch
```

Comment:
Seems like doing `mock_out_backup` as a fixture would have removed the need to add `monkeypatch` where it wasn't otherwise needed; but then you'd have needed to add the backup fixture instead...

Reply:
Right:  I didn't want to make `mock_out_backup` a module-scope fixture (or other large scope), because some of the tests want to mock it out in other ways, and making it a function-scope fixture means adding it to the parameter list (instead of or in addition to `monkeypatch`).